src/app/accounts/register.component.ts

```diff
@@ -65,5 +68,25 @@ export class RegisterComponent extends BaseRegisterComponent {
                 }
             } catch { }
         }
+
+        if (this.policies != null) {
+            this.enforcedPolicyOptions = await this.policyService.getMasterPasswordPolicyOptions(this.policies);
+        }
+    }
+
+    async submit() {
+        if (this.enforcedPolicyOptions == null) {
+            super.submit();
+            return;
+        }
+
+        if (!(this.policyService.evaluateMasterPassword(this.masterPasswordScore, this.masterPassword,
```

Comment:
You dont need the extra parens here.

```suggestion
        if (!this.policyService.evaluateMasterPassword(this.masterPasswordScore, this.masterPassword,
```

Reply:
Updated - only had them in there for my sanity :D